utils/check_repo.py

```diff
@@ -600,21 +600,24 @@ def check_all_models_are_tested():
         raise Exception(f"There were {len(failures)} failures:\n" + "\n".join(failures))
 
 
+def get_models_with_prefix(model: types.ModuleType, attr_prefix: str) -> Set[str]:
```

Comment:
The first argument named `model`, but it is actually a module of auto model. Maybe just name it as `module`.

Reply:
name function as `get_models_in_auto_mappings` might be better.

The prefix is not the prefix of models, but of the auto mappings